I can confirm that it does not like that.  It errors with:

```cpp
[ 29%] thirdparty/embree/common/sys/sysinfo.cpp:299:59: error: too few arguments provided to function-like macro invocation
  299 |     if (nIds >= 7) __cpuid_count(cpuid_leaf_7,0x00000007,0);
      |                                                           ^
/home/tcroc/dev/BlockyBallOT/submodules/godot-src/gitignore/zig/0.13.0-dev.267+793f820b3/zig-linux-x86_64-0.13.0-dev.267+793f820b3/lib/include/cpuid.h:265:9: note: macro '__cpuid_count' defined here
  265 | #define __cpuid_count(__leaf, __count, __eax, __ebx, __ecx, __edx) \
      |         ^
thirdparty/embree/common/sys/sysinfo.cpp:299:20: error: use of undeclared identifier '__cpuid_count'
  299 |     if (nIds >= 7) __cpuid_count(cpuid_leaf_7,0x00000007,0);
      |                    ^
[ 30%] 2 errors generated.
```

Anything else you would like me to try or should we leave it as is?  Should I put a PR upstream to see if they have any feedback?

--- a/thirdparty/embree/common/sys/sysinfo.cpp
+++ b/thirdparty/embree/common/sys/sysinfo.cpp
@@ -295,7 +295,7 @@ namespace embree
     if (nIds >= 1) __cpuid (cpuid_leaf_1,0x00000001);
 #if _WIN32
 #if _MSC_VER && (_MSC_FULL_VER < 160040219)
-#else
+#elif defined(_MSC_VER)
     if (nIds >= 7) __cpuidex(cpuid_leaf_7,0x00000007,0);
 #endif
 #else

--- a/thirdparty/embree/patches/mingw-no-cpuidex.patch
+++ b/thirdparty/embree/patches/mingw-no-cpuidex.patch
@@ -1,0 +1,13 @@
+diff --git a/thirdparty/embree/common/sys/sysinfo.cpp b/thirdparty/embree/common/sys/sysinfo.cpp
+index d01eab3c9d..4ecab05265 100644
+--- a/thirdparty/embree/common/sys/sysinfo.cpp
++++ b/thirdparty/embree/common/sys/sysinfo.cpp
+@@ -295,7 +295,7 @@ namespace embree
+     if (nIds >= 1) __cpuid (cpuid_leaf_1,0x00000001);
+ #if _WIN32
+ #if _MSC_VER && (_MSC_FULL_VER < 160040219)
+-#else
++#elif defined(_MSC_VER)
+     if (nIds >= 7) __cpuidex(cpuid_leaf_7,0x00000007,0);
+ #endif
+ #else

